Tests/SwiftSyntaxBuilderTest/ArrayExprTests.swift

```diff
@@ -47,4 +47,10 @@ final class ArrayExprTests: XCTestCase {
       """
     )
   }
+
+  func testInitializerWithExpressions() {
+    let expressions: [ExprSyntax] = [.init(literal: 0), .init(literal: 1), .init(literal: 2)]
```

Comment:
You could also make this even simpler as follows.

```suggestion
    let expressions: [ExprSyntax] = ["0", "1", "2"]
```

Reply:
nice!